test/e2e/network/util.go

```diff
@@ -76,3 +79,27 @@ func newAgnhostPod(name string, args ...string) *v1.Pod {
 		},
 	}
 }
+
+// VerifySCTPModuleLoadedOnNodes checks whether any node on the list has the
+// sctp.ko module loaded
```

Comment:
This is probably also a good place to document *why* we are doing this:

    // For security reasons, and also to allow clusters to use userspace SCTP implementations,
    // we require that just creating an SCTP Pod/Service/NetworkPolicy must not do anything
    // that would cause the sctp kernel module to be loaded.

Reply:
Fixed.